proposals/records.md

```diff
@@ -38,6 +38,7 @@ Members are synthesized unless a member with a "matching" signature is declared
 an accessible concrete non-virtual member with a "matching" signature is inherited.
 Two members are considered matching if they have the same
 signature or would be considered "hiding" in an inheritance scenario.
+It is an error for a member of a record to be named "Clone".
```

Comment:
Did we really decide to disallow all members named Clone, or should we allow methods named Clone as long as they have parameters?

Reply:
Notes: https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-07-01.md#Confirming-unspeakable-Clone-method-implications

We explicitly discussed methods and fields in LDM ("Clone" fields should be disallowed). For nested types, I chatted with Jared offline and I think we'll want the same behavior.

Reply:
I don't believe that particular variant of the question was raised in LDM. Personally, I'd prefer to disallow _all_ members, just in case we end up wanting to add some kind of parameter list to possibly support only cloning if an initer will actually change a field, but it's a followup we could raise with the LDM if we want to clarify.